### PR TITLE
Maintenance: Remove unneeded variables and localized strings

### PR DESCRIPTION
**Pull request by wutschel**
## Description

This PR removes some unneeded variables and localized strings around `DetailViewController`.

## Summary for release notes

Maintenance: Remove unneeded variables and localized strings

### CODE REVIEW

File: XBMC Remote/DetailViewController.m [hunk 1]
```diff
@@ -839,8 +839,7 @@ - (void)configureLibraryView {
         self.indexView.hidden = YES;
         self.searchController.searchBar.backgroundColor = [Utilities getSystemGray6];
         self.searchController.searchBar.barStyle = UIBarStyleBlack;
-        self.searchController.searchBar.tintColor = tableViewSearchBarColor;
-        searchBarColor = tableViewSearchBarColor;
+        self.searchController.searchBar.tintColor = searchBarColor;
```
Comment (kambala-decapitator) [inline XBMC Remote/DetailViewController.m:842]: is `searchBarColor` set externally? From this commit it looks like it's uninitialised.

Comment (wutschel) [reply XBMC Remote/DetailViewController.m:842]: It is set in `viewDidLoad`:
`searchBarColor = [Utilities get2ndLabelColor];`

Comment (wutschel) [reply XBMC Remote/DetailViewController.m:842]: I just figured I can further reduce variables. Let's keep this open.

Comment (wutschel) [reply XBMC Remote/DetailViewController.m]: Even `searchBarColor` and `collectionViewSearchBarColor` could be removed. I could use #defines for them, if you prefer.

**Comment by kambala-decapitator** [comment]
@wutschel is this ready?

**Comment by wutschel** [comment]
> @wutschel is this ready?

Yes, it is.

**Review by kambala-decapitator — APPROVED** [review]

File: XBMC Remote/DetailViewController.m (hunk 1 above)
Comment (kambala-decapitator) [reply XBMC Remote/DetailViewController.m]: looks already done in the new commit?

Comment (wutschel) [reply XBMC Remote/DetailViewController.m]: Yes, all done in the fixup.

**Comment by wutschel** [comment]
Squashed and rebased to master.

**Pull request merged**